CHANGES.rst

```diff
@@ -4,6 +4,7 @@ New adapter and instrument mechanics
 ------------------------------------
 - Channel class added. Instrument.channels and Instrument.ch_X (X is any channel name) are reserved for channel implementations.
 - All instruments are required to accept a :code:`name` argument.
+- :code:`read_bytes` of all Adapters does not break on a termination character, unless :code:`break_on_termchar` is set to `True`.
```

Comment:
```suggestion
- Changed: :code:`read_bytes` of all Adapters by default does not stop reading on a termination character, unless the new argument :code:`break_on_termchar` is set to `True`.
```

Reply:
I realise I'm still wondering if people would not by default want to read one "bytes message" delimited by the termination char if they use `adapter.read_bytes()` (mirroring `adapter.read()`). You don't think so?

Reply:
We have both use cases for `read_bytes` :

A binary instrument, where you read (certain number of) bytes. Here you do not want to break at a byte which happens to be the (default) term char.

Binary data transfer in am ASCII based instrument: here again you do not want to break.

You have Text based communication, but want to skip the decoding or there are bytes in the message es well...
Here breaking at a term char might be useful.

From these examples I prefer to keep the breaking optional (it is optional in pyvisa as well).

Reply:
Fair enough, thanks!

Reply:
Talking, I forgot the original case (documentation)...